README.md

```diff
@@ -3,406 +3,228 @@
 
-### IDE Plugins
+## Community
 
-Run tests and view test results right from your favorite IDE.
+Follow us on [Twitter](https://twitter.com/DXTestCafe). We post TestCafe news and updates, several times a week.
 
-* [Visual Studio Code](https://github.com/romanresh/vscode-testcafe)
-* [SublimeText](https://github.com/churkin/testcafe-sublimetext)
+## Badge
 
-### Framework-Specific Selectors
+Show everyone you are using TestCafe: ![Tested with TestCafe](https://img.shields.io/badge/tested%20with-TestCafe-2fa4cf.svg)
 
-Work with page elements in a way that is native to your framework.
+To display this badge, add the following code to your repository readme:
 
-* [Aurelia](https://github.com/miherlosev/testcafe-aurelia-selectors)
-* [React](https://github.com/DevExpress/testcafe-react-selectors)
-* [Vue](https://github.com/devexpress/testcafe-vue-selectors)
-
-### Plugins for Task Runners
+```text
+<a href="https://github.com/DevExpress/testcafe">
+    <img alt="Tested with TestCafe" src="https://img.shields.io/badge/tested%20with-TestCafe-2fa4cf.svg">
+</a>
+```
 
-Easily integrate TestCafe into your project's workflow.
+## Contributing
 
-* [Grunt](https://github.com/crudo/grunt-testcafe)
-* [Gulp](https://github.com/DevExpress/gulp-testcafe)
+Report bugs and request features on our [issues page](https://github.com/DevExpress/testcafe/issues).<br/>
+Ask questions and participate in discussions on the [discussion board](https://testcafe-discuss.devexpress.com/).<br/>
+For more information on how to help us improve TestCafe, see the [CONTRIBUTING.md](https://github.com/DevExpress/testcafe/blob/master/CONTRIBUTING.md).
 
-### Browser Providers
+You can use these plugin generators to create your own plugins:
 
-Provide support for more browsers to test against.
+* [Build a browser provider](https://devexpress.github.io/testcafe/documentation/extending-testcafe/browser-provider-plugin/)
+  to set up tests on your on-premises server farm, to use a cloud testing platform, or to start your local browsers in a special way. Use this [Yeoman generator](https://www.npmjs.com/package/generator-testcafe-browser-provider) to write only a few lines of code.
+* To [build a custom reporter](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/)
+  with your formatting and style, check out this [generator](https://www.npmjs.com/package/generator-testcafe-reporter).
 
-* [SauceLabs](https://github.com/DevExpress/testcafe-browser-provider-saucelabs)
-* [BrowserStack](https://github.com/DevExpress/testcafe-browser-provider-browserstack)
-* [Nightmare](https://github.com/ryx/testcafe-browser-provider-nightmare)
-* [iOS Simulator](https://github.com/Ents24/testcafe-browser-provider-fbsimctl)
+If you want your plugin to be listed below, [send us a note in a Github issue](https://github.com/DevExpress/testcafe/issues).
```

Comment:
Probably this link should be - https://github.com/DevExpress/testcafe/issues/new